ci/travis/osx/before_install.sh

```diff
@@ -28,6 +28,7 @@ brew ln expat --force
 brew ln libxml2 --force
 brew ln gettext --force
 brew ln libffi --force
+brew remove gdal
```

Comment:
Seems travis installs it, because when I first tried without removing gdal it crashed all the time.